Add TypeScript Guides (#6454)
DevExpress/devextreme-documentation: concepts/Common/TypeScript Guides/05 Scopes of Types/15 Common Types.md

> Each component includes the necessary types under its alias. However, if multiple components use the same type on the same page, you might get confused about the correct module to import from. In such cases, import the types from the `common` module.

Reviewer: ```suggestion
Each component includes the necessary types under its alias. However, if multiple components use the same type on the same page, it might be unclear which module to import from. In such cases, import the types from the `common` module.
```
